include/mesh/mesh_function.h

```diff
@@ -84,13 +84,17 @@ class MeshFunction : public FunctionBase<Number>,
                 const unsigned int var,
                 const FunctionBase<Number> * master=nullptr);
 
+  /**
+   * A regular copy constructor. Replicates the steps of MeshFunction::clone().
```

Comment:
The copy constructor should not replicate `MeshFunction::clone()`, `MeshFunction::clone()` should just call the copy constructor, e.g.
```
return libmesh_make_unique<MeshFunction>(*this);
```